test/errors/test_FPDF_errors.py

```diff
@@ -21,9 +21,12 @@ def test_encoding_exception():
     pdf = fpdf.FPDF()
     pdf.add_page()
     pdf.set_font("Helvetica", size=15)
-    with pytest.raises(FPDFUnicodeEncodingException):
+    with pytest.raises(FPDFUnicodeEncodingException) as e:
```

Comment:
Could you rename this variable into `error`, which is a lot more explicit, please?